tooling/components/io.siddhi.distribution.editor.core/src/main/java/io/siddhi/distribution/editor/core/internal/ExportUtils.java

```diff
@@ -301,6 +304,13 @@ public File createZipFile() throws DockerGenerationException, KubernetesGenerati
         String content = new String(data, StandardCharsets.UTF_8);
         String productVersion = this.getConfigurations().getProductVersion();
         content = content.replaceAll(PRODUCT_VERSION_TEMPLATE, productVersion);
+
+        if (exportType != null && exportType.equals(EXPORT_TYPE_KUBERNETES)) {
```

Comment:
Change this to EXPORT_TYPE_KUBERNETES.equals(exportType)

Reply:
Fixed in https://github.com/siddhi-io/distribution/pull/794/commits/299f71bb802e9cd7c0b16f595f2247026f0b1739